alphagov/paas-cf
# [#168427527] Credhub for cf secrets cleanup

What
----

⚠️  DO NOT MERGE ⚠️ 

Follow up to #2083 to remove the now-unnecessary handling of cf-secrets credentials from S3 (they're managed in credhub now)

How to review
-------------

* Code review
* Tell everyone to run #2083 through their dev environments
* Run `aws s3 rm "s3://gds-paas-${DEPLOY_ENV}-state/cf-secrets.yml"` to remove the old cf-secrets from your dev env
* Run this branch down your dev env
* Run `aws s3 rm "s3://gds-paas-${DEPLOY_ENV}-state/cf-secrets.yml"` to remove the old cf-secrets from staging and prod
* Merge this and make sure it deploys cleanly

Who can review
--------------

Not @richardtowers
